spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+describe UsersController do
+  describe "#index" do
+    it "responds successfully" do
+      get users_path
+      expect(response).to be_successful
+    end
+
+    it "returns a 200 response" do
```

Comment:
it文なのですが、200だけじゃなく200レスポンスの説明文も書いてあげるとより親切ですね！
```suggestion
    it "returns 200 OK" do
```